PF/PF-Assignment-1/Shopping.java

```diff
@@ -0,0 +1,132 @@
+package sc;
+import java.util.ArrayList;
+import java.util.List;
+public class Shopping {
+       
+    private List<Item> Cart = new ArrayList<Item>();
+    
+    ShoppingCart(){
+    	
+    }
+ 
+    public void addItem(String Pname,float Pprice,int Pquantity )
```

Comment:
Follow Java naming conventions while naming the methods, class, data members, variables.
Like here, variables always start with lower case. It should be renamed as 
public void addItem(String pName, float pPrice, int pQuantity)